### DISCUSSION TITLE
What are the production canister IDs for the `identity_manager` and `pub_sub_channel` canisters?

### DISCUSSION THREAD

**Discussion by paulyoung** [internet-identity-labs/NFID-SDK#141] [Q&A] [answered]
Which canister IDs should I use to set up the `identity_manager` and `pub_sub_channel` as [remote canisters](https://internetcomputer.org/docs/current/developer-docs/updates/release-notes/0.9.2-rn#new-feature-remote-canister-support)?

Some context: I prefer to install any third-party canisters as part of the same build step as my own canisters. For Internet Identity, this means I have the following in my `dfx.json`:

```json
    "internet_identity": {
      "type": "custom",
      "wasm": "result/remote/internet_identity/internet_identity_dev.wasm",
      "candid": "result/remote/internet_identity/internet_identity.did",
      "remote": {
        "id": {
          "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
        }
      }
```

I'd like to do the same for NFID but don't know the canister IDs. For this use case it seems unnecessary so I raised that here: https://github.com/dfinity/sdk/issues/2216#issuecomment-1211193646

**Edit:** Perhaps this approach isn't really feasible since the `nfid` frontend still needs to be served by webpack.

**Comment by paulyoung** [comment]
I think I can probably use the ["install arbitrary Wasm into canisters"](https://github.com/dfinity/sdk/blob/6be3ac3f6927c0d252db2688856dc3d64b7a4547/CHANGELOG.md#feat-install-arbitrary-wasm-into-canisters) feature to do this instead.

**Reply by paulyoung** [reply]
This doesn't help since the dfx config is still needed.

**Comment by dostro** [comment, marked as answer]
@paulyoung 
`identity_manager`: `bryxc-zqaaa-aaaal-aajga-cai`
`pub_sub_channel`: `iyvpb-uqaaa-aaaag-qabxa-cai`

But all the requests go through our frontend because just about all calls must be authenticated - is your goal to streamline local development?

**Reply by paulyoung** [reply]
> all the requests go through our frontend because just about all calls must be authenticated

Understood. For cases like II and NFID it seems redundant to have to provide this.

> is your goal to streamline local development?

Exactly.
